[EDU-3217] refactor: update Upstash Geolocation template guide. (#468)
aziontech/docs: src/content/docs/en/pages/guides/marketplace/templates/upstash-geolocation/upstash-geolocation.mdx

---

> title: How to use the Upstash GeoLocation template through Azion

Author: ```suggestion
title: How to use Upstash GeoLocation through Azion
```

Preview
</Badge>

> The **Upstash GeoLocation** template is an automation designed to deploy a database directly on the edge while enabling the configuration of custom messages that will be delivered according to the user's geographical location.

Author: ```suggestion
**Upstash GeoLocation** is an automation designed to deploy a database directly on the edge while enabling the configuration of custom messages that will be delivered according to the user's geographical location.
```

### Key configurations

> Upstash GeoLocation EdgeDeploy creates a new Azion edge application and its domain. It also creates an edge function to provide the arguments for the geolocation, a new repository in your GitHub account based on a [public template](https://github.com/aziontech/azion-samples/tree/dev/templates/upstash-geolocation), and a static interface to exhibit the messages.

Reviewer: ```suggestion
Upstash GeoLocation EdgeDeploy creates a new Azion edge application and its domain. It also creates an edge function to provide the arguments for the geolocation, a new repository in your GitHub account based on a [public template](https://github.com/aziontech/azion-samples/tree/dev/templates/upstash-geolocation), and a static interface to show the messages.
```

<Badge>
[Edge Functions](/en/documentation/products/edge-application/edge-functions/)
</Badge>
<Badge>
[Application Acceleration](/en/documentation/products/edge-application/application-acceleration/)
</Badge>

> You must activate this product separately via RTM. To do so:

Author: ```suggestion
You must activate these modules separately via RTM. To do so:
```
